### PR TITLE
added the edit order page and made a route for it

### PR DESCRIPTION
**Pull request by ShivaKakarla512**
No styling yet - WIP

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -25,7 +25,8 @@
     ],
     "rules": {
         "indent": ["error", "tab"],
-        "no-tabs": "off"
+        "no-tabs": "off",
```
Comment (dhee9000) [inline .eslintrc:28]: let's remove any eslintrc changes from PRs. let's use unix line endings, you should be able to change VSCode settings to edit with unix line endings too from the bottom right

File: package.json [hunk 2]
```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@coreui/coreui": "^4.0.0-rc.5",
```
Comment (dhee9000) [inline package.json:6]: are we using the bootstrap version of coreui too? I think all components are also accessible from the React package but let me know if there is actually a dependency on the bootstrap package.

File: src/routes/index.js [hunk 3]
```diff
@@ -23,6 +24,12 @@ const ROUTES = [
 			},
 		],
 	},
+	{
+		path: '/editOrder',
```
Comment (dhee9000) [inline src/routes/index.js:28]: let's make the URL structure for this be /orders/<orderId>/edit. Ignore the <orderId> for now and just show the page for any orderId value

File: src/views/editOrderFields/EditOrderFields.tsx [hunk 4]
```diff
@@ -0,0 +1,216 @@
+import React from 'react';
+import './EditOrderFields.css';
+import '@coreui/coreui/dist/css/coreui.min.css';
```
Comment (dhee9000) [inline src/views/editOrderFields/EditOrderFields.tsx:3]: Does this do anything that the React import doesn't do?

File: src/views/editOrderFields/EditOrderFields.tsx [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+	CContainer,
+	CRow,
+	CCol,
+	CFormSelect,
+	CButton,
+	CFormInput,
+	CFormTextarea,
+} from '@coreui/react';
+
+const documentType = [
```
Comment (dhee9000) [inline src/views/editOrderFields/EditOrderFields.tsx:19]: lets move all of these constants into a separate file called something like EditOrderFieldsConfig.js or something like that (up to you if you have a better name).

File: yarn.lock [hunk 6]
```diff
@@ -1151,6 +1151,18 @@
     exec-sh "^0.3.2"
     minimist "^1.2.0"
 
+"@coreui/coreui@^4.0.0-rc.5", "@coreui/coreui@^4.0.1":
```
Comment (dhee9000) [inline yarn.lock:1154]: again same as package.json but if that updates so will this.

**Review by dhee9000 — CHANGES_REQUESTED** [review]
Reviewed the code, looks great! There are a couple minor changes that I pointed out, fix and commit and it looks good.

**Review by koletimanish — APPROVED** [review]
Looks good to me!

**Review by dhee9000 — APPROVED** [review]
Looks great! Ready to merge into develop.

**Pull request merged**